test/api/run

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
```

Comment:
I can't find any obvious error in this code, but I want to point out a second time in two days that this is yet another copy of the same code, which I am still suggesting to you to not do, and instead use the python library I wrote to do the same thing - see example in test/alternator/run. This is exactly why I wrote that Python library that both test/alternator/run and test/cql-pytest now run - so we won't have all that code duplication, and every time we need to fix a bug or add a boot-time parameter, we won't need to fix it in 7 places.

Reply:
Sorry, I followed you references to test/redis/run in https://github.com/scylladb/scylla/pull/9731
I'll convert it to Python and use the requests Python module instead of curl.

Reply:
Thanks. I'll also do what I preach and convert redis/run to Python today (https://github.com/scylladb/scylla/issues/9748).
To be honest, this is not my preaching - I actually liked those self-contained shell scripts when there were only one or two copies, until @kostja and others rightly asked to convert it to Python (although that is just a matter of preference) and move the main scylla-starting logic to one place instead of being duplicated in multiple scripts.

Reply:
I sent a patch titled "test/redis: replace run script with shorter Python script"
You can do exactly the same, I think it's very easy, or even easier (you don't even need the new "redis_cmd" because you don't even need to change Scylla's parameters - the REST API is on by default). The code is certainly much shorter than the shell script with the full logic, Scylla command line, etc.